### PR TITLE
Fix in dev classpath issue

### PR DESCRIPTION
**Pull request by thiakil**
Adds indev launch handlers that will add folder paths from the system classpath to the cpw launcher cp

use `--launchTarget omldevclient` / `--launchTarget omldevserver`

### CODE REVIEW

File: src/main/java/com/openmodloader/loader/launch/OMLDevClientLaunchHandler.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class OMLDevClientLaunchHandler extends OMLClientLaunchHandler {
+
+	@Override
+	public Path[] identifyTransformationTargets() {
+		Path[] basePaths = super.identifyTransformationTargets();
+		List<Path> paths = new ArrayList<>(Arrays.asList(basePaths));
```
Comment (Gegy) [inline src/main/java/com/openmodloader/loader/launch/OMLDevClientLaunchHandler.java:13]: Very tiny thing, does this need to be wrapped in an ArrayList?

Comment (thiakil) [reply src/main/java/com/openmodloader/loader/launch/OMLDevClientLaunchHandler.java:13]: Nah, just a way to combine the elements. If you can think of a better way go for it

Comment (Gegy) [reply src/main/java/com/openmodloader/loader/launch/OMLDevClientLaunchHandler.java:13]: Oh, I mean could it not just be `Arrays.asList(basePaths)`?

Comment (thiakil) [reply src/main/java/com/openmodloader/loader/launch/OMLDevClientLaunchHandler.java:13]: I don't believe you can add elements to those lists... They're a special wrapper

**Comment by Gegy** [comment]
Thanks! :D

**Pull request merged**